app/views/events/show.html.erb

```diff
@@ -5,6 +5,7 @@
 <%= render_component "event", context: :page, event: @event, primary_neighbourhood: @primary_neighbourhood %>
 
 <div class="c c--narrow c--space-after event__fullinfo">
+  <%= event_link(@event) %>
```

Comment:
put below description

Reply:
ha it was below but i thought as a CTA it would stand out more as the first thing. will move